app/src/main/java/eu/niggas_with_attitude/qrify/ScanResultActivity.java

```diff
@@ -45,5 +53,14 @@ protected void onCreate(Bundle savedInstanceState) {
                 startActivity(i);
             });
         }
+
+        copyButton.setOnClickListener(
+            view -> {
+                clipData = ClipData.newPlainText("text", message);
+                clipboardManager.setPrimaryClip((clipData));
```

Comment:
By local variable I meant this:
```suggestion
            view -> {
                ClipData clipData = ClipData.newPlainText("text", message);
                clipboardManager.setPrimaryClip(clipData);
```